index.html

```diff
@@ -484,14 +484,14 @@ <h3>Media Feed Item</h2>
                 numberOfSeasons</a> property MUST be a positive integer.
               </li>
               <li>
-                The <a data-cite="schema-org/episode#">episode</a> property MAY
-                be present. If it is, it should contain a valid <a>media item
-                episode</a>.
+                The <a data-cite="schema-org/episode#">episode</a> property supports
+                <a>multiple values</a> and if present it MUST contain valid
```

Comment:
Here and elsewhere, what do you think about saying "if present it MUST contain one or two valid media item episodes."

Reply:
See above

Reply:
If we hardcode the requirement that it can only be one or two then we run the risk above of old versions of a browser not working with sites using newer versions of the api.